components/src/Tables/Table.js

```diff
@@ -1,38 +1,98 @@
 import React from 'react';
 import styled from 'styled-components';
+import { colour, space, font, radius, shadow } from '@nulogy/tokens';
+import QuietButton from '../Button/QuietButton';
 
-const borderStyle = ({ theme }) => `1px solid ${theme.colour.neutral[400]}`;
+const borderWidth = 1;
+
+const withoutBorder = value => value - borderWidth;
+
+const lineHeight = name => font.lineHeight[name] * font.size.medium;
 
 export const Cell = styled.td`
-  padding: 1rem;
-  text-align: left;
-  border-bottom: ${borderStyle};
+  font-size: ${font.size.small}px;
+  line-height: ${lineHeight('smaller') / font.size.small};
```

Comment:
Do we actually want to set the font-size to be small and set a tighter line-height on tables? 

Quite a bit of the content in PackManager is tabular. This will have the effect of negating a lot of the roominess we were hoping to bring to the app.

Reply:
I picked small because regular looked way to different from the current state. Currently table font size is 12px. I agree we should address this in the next release but for the 21st I would keep it as is.